src/commands/test/asserts/asserts_test.py

```diff
@@ -0,0 +1,18 @@
+from os import path
+from pathlib import Path
+
+import pytest
+
+from src.commands.test import TestRunnerWithCheatcodes
+
+CURRENT_DIR = Path(__file__).parent
+
+
+@pytest.mark.asyncio
+async def test_asserts():
+    runner = TestRunnerWithCheatcodes(
+        include_paths=[path.join(CURRENT_DIR, "..", "..", "..", "..", "cairo")]
```

Comment:
can we use pathlib for paths manipulation everywhere?

Reply:
We can, but maybe not in this PR. I'd wait for https://github.com/software-mansion/protostar/pull/76 to be merged.